Detector/DetFCChhBaseline1/compact/FCChh_MuonSystemSimple.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is no longer an envelope but preliminary detector concept so it should go into dedicated `Detector/DetFCChhMuonSimple` directory.
Please also include then that file in `Detector/DetFCChhBaseline1/compact/FCChh_DectMaster.xml` so the overlaps are checked.